app/renderer/lib/appium-client.js

```diff
@@ -327,8 +330,11 @@ export default class AppiumClient {
             const deviceScreenInfo = await this.driver.executeScript('mobile:deviceScreenInfo', []);
             webviewLeftOffset = deviceScreenInfo.statusBarSize.height;
           } catch (e) {
-            // in case driver does not support mobile:deviceScreenInfo
-            webviewLeftOffset = sessionDetails.statBarHeight;
+            try {
+              const sessionDetails = await this.driver.getSession();
```

Comment:
should we memoize this so we don't make the network call twice?

Reply:
Not necessary since these are either Android or iOS, so basically it occurs once

Reply:
It will still be called on every refresh if using hybrid mode - but now only in a few specific cases.